apple-nav/src/components/MainNav.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+const MainNav = props => {
+  //   console.log(props);
+  return (
+    <nav className="main-nav">
+      {props.data.map((item, i) => (
+        <Link to={`/${item.name}`} key={i}>
```

Comment:
Great job making your links dynamic. They are all working and take me to the individual image.